js/src/ui/Form/TypedInput/typedInput.js

```diff
@@ -61,8 +67,11 @@ export default class TypedInput extends Component {
   };
 
   componentWillMount () {
-    if (this.props.isEth && this.props.value) {
-      this.setState({ isEth: true, ethValue: fromWei(this.props.value) });
+    const { isEth, value } = this.props;
+
+    if (typeof isEth === 'boolean' && value) {
```

Comment:
Well it's more or less like `allowCopy` : could be true or false whether to show it or not, or the value to actually copy. I agree that it might not be the best, but I'm not sure that it's in the scope of this PR.